### PR TITLE
rebalance deathrattle serpent

### PR DESCRIPTION
**Pull request by sonphantrung**
#### Summary
Balance "rebalance deathrattle serpent"

#### Purpose of change
From my old Discord memories, there came up a conversation about how STUPIDLY POWERFUL the deathrattle serpent is: 90kg, 70L (much lower than hulks), but the damage is `8d8 bash`. HOW OP IT IS!

#### Describe the solution
Rebalanced the serpent: `6d8 bash + 12 cut` now.

#### Describe alternatives you've considered
Not doing this

#### Testing
Should be good ;)

### CODE REVIEW

**Comment by Nebnis** [comment]
I think it should be lowered even further. A hulk got 4d8 for reference, so it should be that or lower maybe increase cut again. The snake already have a deadly poison and rattlesnakes are not known for being constrictors.

**Review by MNG-cataclysm — APPROVED** [review]

File: data/json/monsters/reptile_amphibian.json [hunk 1]
```diff
@@ -284,9 +284,9 @@
     "morale": 100,
     "aggro_character": false,
     "melee_skill": 9,
-    "melee_dice": 8,
+    "melee_dice": 6,
     "melee_dice_sides": 8,
-    "melee_damage": [ { "damage_type": "cut", "amount": 8 } ],
+    "melee_damage": [ { "damage_type": "cut", "amount": 12 } ],
```
Comment (Maleclypse) [inline data/json/monsters/reptile_amphibian.json:289]:
```suggestion
    "melee_damage": [ { "damage_type": "stab", "amount": 8 } ],
```

File: data/json/monsters/reptile_amphibian.json [hunk 2]
```diff
@@ -284,9 +284,9 @@
     "morale": 100,
     "aggro_character": false,
     "melee_skill": 9,
-    "melee_dice": 8,
+    "melee_dice": 6,
     "melee_dice_sides": 8,
```
Comment (Maleclypse) [inline data/json/monsters/reptile_amphibian.json:288]:
```suggestion
    "melee_dice_sides": 6,
```

File: data/json/monsters/reptile_amphibian.json (hunk 1 above)
Comment (Maleclypse) [reply data/json/monsters/reptile_amphibian.json:289]: These bites should probably be stab instead of cut so I've shrunk their damage output.

**Review by Maleclypse — APPROVED** [review]

**Review by LyleSY — APPROVED** [review]
Love it, puts this monster's damage within the range in the balance doc, fits under "Very High Damage" https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/GAME_BALANCE.md

**Pull request merged**